src/applications/appeals/996/tests/containers/ConfirmationPage.unit.spec.jsx

```diff
@@ -91,4 +91,26 @@ describe('Confirmation page', () => {
     expect(tree.find('li .dd-privacy-hidden').length).to.eq(1);
     tree.unmount();
   });
+
+  it('should render with no data', () => {
```

Comment:
Oh is this because the data goes away once it's submitted?

Reply:
No, this is testing all the branches in the code. If you reload the confirmation page, there won't be any data, but it should still render

Reply:
Ah got it. TY